ignite/metrics/accuracy.py

```diff
@@ -161,11 +163,12 @@ def update(self, output: Sequence[torch.Tensor]) -> None:
             y = torch.transpose(y, 1, last_dim - 1).reshape(-1, num_classes)
             correct = torch.all(y == y_pred.type_as(y), dim=-1)
 
-        self._num_correct += torch.sum(correct).item()
+        # Don't need to detach here because torch.eq is not differentiable, so the computation graph is detached anyway.
```

Comment:
Let's remove this comment and call detach on the output as in other classes:
```python
# y_pred, y = output
y_pred, y = output[0].detach(), output[1].detach()
```

Reply:
Sounds good!